words/disasm.asm

```diff
@@ -135,15 +128,10 @@ _byte_loop:
 
                 ; We have a three-byte instruction, so we need to get the MSB
                 ; of the operand. Move to the next byte
-                inc 4,x
-                bne +
-                inc 5,x                 ; ( addr+2 u-1 LSB )
-+
-                lda 2,x
-                bne +
-                dec 3,x
-+
-                dec 2,x                 ; ( addr+2 u-2 LSB )
+                jsr w_not_rot           ; ( LSB addr u )
```

Comment:
some of these changes are only a little shorter and obviously much slower, but a lot easier to read which I figured was reasonable tradeoff

Reply:
I'm not super concerned about the speed during disassembly, so readability is probably the better optimization target here.